eng/pipelines/stress/http-linux.yml

```diff
@@ -0,0 +1,34 @@
+trigger: none
```

Comment:
Do we want 1 build definition for windows and one for linux? Or should 1 build just include both as separate jobs?

Reply:
I think it's a good idea to trigger separately, on demand.